projects/observability/src/shared/components/topology/topology.component.ts

```diff
@@ -58,9 +59,18 @@ export class TopologyComponent implements OnChanges, OnDestroy {
   @Input()
   public shouldAutoZoomToFit?: boolean = false;
 
+  @Input()
+  public draggableNodes?: boolean = true;
+
   @Input()
   public layoutType?: TopologyLayoutType;
 
+  @Input()
+  public customLayout?: TopologyLayout; // This will override `layoutType` property
+
+  @Input()
+  public supportGroupNode?: boolean;
```

Comment:
Do we need this as an input or can  we extract this info from our custom group node layout property?

Reply:
The layout and group nodes are different IMO. The current flow looks like this.

1. We create group nodes like we create entity nodes using the data that we get from the backend.
2. We pass this data to the topology.
3. Now topology handles group nodes based on the property.
4. custom layout (with some business logic) is passed from the parent component and used to position the nodes on some layout logic. The layout may or may not care about the group nodes.

The one thing that we can do here is we can extract this property by checking if the nodes contain a group node or not. If this works I can make the changes